src/main.js

```diff
@@ -31,19 +35,35 @@ const auth = require('./auth');
  */
 async function main() {
   try {
-    const allArgs = process.argv;
-    if (allArgs.length > 3) {
-      throw new Error(
-          'Incorrect number of arguments. User can only specify the path to npmrc file.');
-    }
-    let configPath = __dirname + '/.npmrc';
-    if (allArgs.length == 3) {
-      configPath = allArgs[2];
-    }
+    const allArgs = yargs(hideBin(process.argv))
+      .command('$0 [config]', 'Refresh the tokens for .npmrc config file', (yargs) => {
+        yargs.positional('config', {
+          type: 'string',
+          describe: '(Deprecated) Path to the .npmrc file to update auth tokens'
+        })
+      })
+      .option('project-config', {
+        type: 'string',
```

Comment:
@nyap @lindsayismith I decided to call the the flags `project-config` and `user-config` during implementation while in the docs there were called `from-config` and `to-config`. These two feel more straightforward from the user perspective, but I can change them back if you think `from/to` are better.

Reply:
Another alternative would be `repo-config` and `credential-config`. The flags describe  what settings we are writing instead of where we are writing them, and we expect you'd only use these flags when you want to choose a location other than our defaults (.npmrc in current dir and .npmrc in user dir).

Reply:
This sounds nice. Updated.